client/packages/core/src/Reactor.js

```diff
@@ -292,25 +294,37 @@ export default class Reactor {
 
   /**
    * @param {'enqueued' | 'pending' | 'synced' | 'timeout' |  'error' } status
-   * @param string eventId
-   * @param {{message?: string, hint?: string, error?: Error}} [errDetails]
+   * @param {string} eventId
+   * @param {{message?: string, type?: string, status?: number, hint?: unknown}} [errorMsg]
```

Comment:
Aside, do we actually know that all callsites to this truly send message,type,status,hint? May be good to double check